core/src/main/java/uk/gov/justice/services/core/dispatcher/DispatcherDelegate.java

```diff
@@ -58,6 +50,17 @@ public JsonEnvelope requestAsAdmin(final JsonEnvelope envelope) {
         return response;
     }
 
+    @Override
+    public <T> Envelope<T> requestAsAdmin(final Envelope<?> envelope, final Class<T> clazz) {
+        final Envelope<JsonValue> jsonValueEnvelope = envelopePayloadTypeConverter.convert(envelope, JsonValue.class);
```

Comment:
is this bit always the same, the conversion and repacking>? could we make that a private method, then we do the dispatch/as admin in the top level method as required?